.github/workflows/README.md

```diff
@@ -114,6 +115,82 @@ jobs:
       github-token: "${{ secrets.GH_TOKEN }}"
 ```
 
+## Kustomize Release
+
+This workflow will deploy to GKE using a Kustomize root directory.
+
+### Prerequisites
+
+Set up GitHub pages for your repository in Settings → Pages → Build and deployment source → GitHub Actions. A special `gh-pages` branch is not needed, since we will use GitHub actions to deploy a Pages artifact.
+
+Currently it is not possible to download a previously created Pages artifact as they quickly expire after deploying it. When releasing an update to a Helm chart, we want to keep all previous versions of the Helm chart available. Therefore, as a workaround, we download the index.yaml file from Pages, parse all referenced releases, and download these .tgz packages from Pages as well. Then we package the new version and update the index. Afterwards, a new Pages artifact is created from these files and finally deployed.
```

Comment:
We don't need a github pages branch for the kustomize action to work. This was for publishing Helm artifacts. You can remove the whole `Prerequisites` section